dipy/workflows/tests/test_align.py

```diff
@@ -20,12 +31,127 @@ def test_reslice():
+                                         out_quality='trans_q.txt')
+
+            dist = read_distance('trans_q.txt')
+            npt.assert_equal('%.2f' % dist, '%.2f' % -0.3953547764454917)
+            npt.assert_equal(os.path.exists(out_moved), True)
+            npt.assert_equal(os.path.exists(out_affine), True)
+
+        def test_rigid():
+
+            out_moved = pjoin(temp_out_dir, "rigid_moved.nii.gz")
+            out_affine = pjoin(temp_out_dir, "rigid_affine.txt")
+
+            image_registeration_flow.run(static_image_file,
+                                         moving_image_file,
+                                         transform='rigid',
+                                         out_dir=temp_out_dir,
+                                         out_moved=out_moved,
+                                         out_affine=out_affine,
+                                         save_metric=True,
+                                         level_iters=[100, 10, 1],
+                                         out_quality='rigid_q.txt')
+
+            dist = read_distance('rigid_q.txt')
+            npt.assert_equal('%.2f' % dist, '%.2f' % -0.6900534794005155)
+            npt.assert_equal(os.path.exists(out_moved), True)
+            npt.assert_equal(os.path.exists(out_affine), True)
+
+        def test_affine():
+
+            out_moved = pjoin(temp_out_dir, "affine_moved.nii.gz")
+            out_affine = pjoin(temp_out_dir, "affine_affine.txt")
+
+            image_registeration_flow.run(static_image_file,
+                                         moving_image_file,
+                                         transform='affine',
+                                         out_dir=temp_out_dir,
+                                         out_moved=out_moved,
+                                         out_affine=out_affine,
+                                         save_metric=True,
+                                         level_iters=[100, 10, 1],
+                                         out_quality='affine_q.txt')
+
+            dist = read_distance('affine_q.txt')
+            npt.assert_equal('%.2f' % dist, '%.2f' % -0.7670650775914811)
+            npt.assert_equal(os.path.exists(out_moved), True)
+            npt.assert_equal(os.path.exists(out_affine), True)
+
+        # Creating the erroneous behavior
+        def test_err():
+
+            npt.assert_raises(ValueError, image_registeration_flow.run,
+                              static_image_file,
+                              moving_image_file,
+                              transform='notransform')
+
+            npt.assert_raises(ValueError, image_registeration_flow.run,
+                              static_image_file,
+                              moving_image_file,
+                              metric='wrong_metric')
+
+        test_com()
+        test_translation()
+        test_rigid()
+        test_affine()
+        test_err()
```

Comment:
@nilgoyette: Thank again for this suggestion. Here I am a bit confused, please correct me if I am wrong. From what I understand is that during Travis checks, it automatically looks for all the files starting from 'test_' and executes them. I had that syntax (that you have suggested) earlier but then I removed it because I came to know that I don't need it for the test to execute.

However, I will definitely look into it if changing this one will help in both running the test independently and also in the travis checks.

Thanks again.

Reply:
You're right, Travis and other tools don't need the `run_module_suite()`. However, it's practical for us when we run this file in command line or with the play button in our favorite IDE.

Reply:
Using run_module_suite() now. Thanks.

Reply:
Good thing you're using `run_module_suite()` but why are you still calling `test_com()`, `test_translation()`, etc.?

Reply:
I was confused too @nilgoyette but when you look at the indentation, these calls are inside `test_image_registration()` function and call local functions.